.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: ruby
 rvm:
-  - 2.2.4
-before_install: gem install bundler -v 1.11.2
+  - 2.5.3
```

Comment:
Can we delete this file and get rid of TravisCI? (This is a question, I don't know how travis is set up for this)

Reply:
TravisCI says this is "not an active repository" and there are no branches/pulls/builds. I'll remove this.